src/main/resources/com/rsmart/kuali/kfs/cr/document/datadictionary/CheckReconciliationMaintenanceDocument.xml

```diff
@@ -67,6 +67,9 @@
               p:required="false" />
         <bean parent="MaintainableFieldDefinition"
               p:name="versionNumber" />
+        <bean parent="MaintainableFieldDefinition"
+              p:name="active"
+              p:defaultValue="true" />

```

Comment:
Are these changes part of KFSPTS-30132? I don't think this file should be in this PR.